doc/source/how-to-upgrade-to-flower-next.rst

```diff
@@ -210,6 +264,30 @@ Deployment
         --ssl-certfile <your-server-cert-filepath> \
         --ssl-keyfile <your-privatekey-filepath>
 
+    # Flower v1.13
+    # In a new terminal window, start a long-running SuperNode
+    $ flower-supernode \
+         --superlink 127.0.0.1:9092 \
+         --node-config "partition-id=0 num-partitions=2" \
+         --supernode-address 127.0.0.1:9094 \
+         --isolation subprocess \
+         --root-certificates <your-ca-cert-filepath>
+
+    # In another terminal window, start another long-running SuperNode (at least 2 SuperNodes are required)
+    $ flower-supernode \
+         --superlink 127.0.0.1:9092 \
+         --node-config "partition-id=1 num-partitions=2" \
+         --supernode-address 127.0.0.1:9095 \
+         --isolation subprocess \
+         --root-certificates <your-ca-cert-filepath>
+
+    # In a new terminal, start the SuperExec process with the following command:
+    $ flower-superexec \
+        --executor flwr.superexec.deployment:executor \
+        --executor-config 'superlink="127.0.0.1:9091"' \
+        --root-certificates <your-ca-cert-filepath>
```

Comment:
See above comment.

Reply:
Thank you for pointing this out. Removed.